build.gradle

```diff
@@ -104,6 +104,38 @@ task exportSampleSphereBlast {
     }
 }
 
+task buildTestApp {
+    description 'Creates the instant app test project, then builds a test app'
+    def buildAndRunFunction = 'GooglePlayInstant.Samples.TestApp.Editor.TestAppBuilder.Build'
```

Comment:
Remove "andRun". Or just inline this constant.

Reply:
Done